src/core/io/tests/4C_io_input_spec_test.cpp

```diff
@@ -637,10 +664,10 @@ required: true
   - name: e
     type: selection
     required: false
-    default: 1
+    default: e1
     choices:
-      e1: 1
-      e2: 2
+      - name: e1
+      - name: e2
```

Comment:
FYI @gilrrei: `default` is now correctly printed as the value you need to enter and I removed the internal values from the choices.